regression-test/suites/query_p0/dual/dual.groovy

```diff
@@ -0,0 +1,70 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite('dual') {
+
+    qt_sql 'select 1 from dual'
+    qt_sql 'select 1 from dual where 1'
+    qt_sql 'select 1 from dual where 1 = 1'
+    qt_sql 'select 1 from dual where 0'
```

Comment:
please add more complex test case. such as in cte, in subquery, in set operand, use with group by, window and other syntax